### ISSUE TITLE
CRTs: allow anyone to trigger closing revenue share

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#5177]
# Problem
People start a revenue share, it ends but they never place the closing TX which means it is impossible for the CRT to return to the market.

# Solution
Allow for the runtime to let any account trigger the close revenue share TX--there doesn't seem to be any malicious action possible with this and the DAO could set up an account that does it automatically up to some certain scale.

**Comment by bedeho** [comment]
Yeah, I just became aware of this today, I guess this was a problem with the way we chose to solve the problem we found, def. could have been done more practically, but allowing anyone to close is not needed, it can happen automatically.
